aptos-move/move-examples/defi/aptoswap/core/sources/utilities.move

```diff
@@ -0,0 +1,20 @@
+module pool_manager::utilities {
+    use aptos_framework::object::Object;
+    use aptos_framework::fungible_asset::Metadata;
+    use std::string::String;
+    use aptos_framework::fungible_asset;
+    use std::string;
+
+    friend pool_manager::liquidity_pool;
+
+    public(friend) fun swap_lp_token_name_and_symbol(x: Object<Metadata>, y: Object<Metadata>): (String, String) {
+        let name = fungible_asset::name(x);
```

Comment:
You can now use the cool functions in aptos_std::string_utils to do all this more easily